src/content/docs/new-relic-solutions/new-relic-solutions/plan-your-cloud-adoption/optimize-customer-experience.mdx

```diff
@@ -394,8 +394,4 @@ With just a few setup steps, you can [get your monitors running](#deploy-synthet
 
 Check out the [Introduction to New Relic One dashboards](/docs/dashboards/new-relic-one-dashboards/get-started/introduction-new-relic-one-dashboards) which shows you many examples of these kinds of queries, using data from your own account. You can quickly get a feel for how to start building queries, widgets, and dashboards.
 
-After you identify your SLAs and KPIs, Browser and Mobile monitoring let you quickly start collecting data from your webpages and mobile apps, and Synthetics lets you schedule the monitors you need so that you can find out about problems before your customers start telling you about them. Bringing it all together in dashboards, you can start building the before/after perspective you need to prove the value of moving to the cloud and gain information that will help you optimize your customer’s experience.
-
-## For more help [#more_help]
-
-Explore the [New Relic Platform](https://newrelic.com/products).
+After you identify your SLAs and KPIs, Browser and Mobile monitoring let you quickly start collecting data from your webpages and mobile apps, and Synthetics lets you schedule the monitors you need so that you can find out about problems before your customers start telling you about them. Bringing it all together in dashboards, you can start building the before/after perspective you need to prove the value of moving to the cloud and gain information that will help you optimize your customer’s experience.
```

Comment:
I don't think Browser and Mobile s/be initial cap any more? "After you identify your SLAs and KPIs, our browser and mobile monitoring capabilities let you..." ?

Reply:
I _think_ I fixed this in my other PR for browser, but i'll change here JIC